### DISCUSSION TITLE
Rendering of a way highway=path + access=private

### DISCUSSION THREAD

**Discussion by demonbox73** [osmandapp/OsmAnd#14293] [Maps Related]
Hi everyone, I noticed that the rendering of a way
highway=path
access=private
it's the same with or without the access=private

this happens with the style map OsmAnd.

In my opinion this is not correct as a path with private access should be marked differently from a path that has free access. 
Do you agree?

Thanks very much.
--enrico

**Comment by njohnston** [comment]
Go to "Configure Map", select "Details" and select "Show access restrictions and toll". Ways with `access=private` will be shown differently (red dots along them).

**Comment by demonbox73** [comment]
oh... yes perfect! 
how did I not think about it ??
thanks very much @njohnston 
;-))
--enrico
